src/data/boxes.json

```diff
@@ -273,6 +273,14 @@
 			"arbitrum"
 		]
 	},
+	"snap-box": {
+		"description": "Metamask Snap box",
+		"stars": 4,
+		"tags": [
+			"metamask",
+			"snap"
```

Comment:
Can this be "snaps" instead? Snaps is the API

Reply:
For sure, done (2d3911eb)